templates/repo/projects/list.tmpl

```diff
@@ -2,41 +2,39 @@
 <div role="main" aria-label="{{.Title}}" class="page-content repository projects milestones">
 	{{template "repo/header" .}}
 	<div class="ui container">
-		<div class="navbar">
-			{{template "repo/issue/navbar" .}}
-			{{if and .CanWriteProjects (not .Repository.IsArchived)}}
-				<div class="ui right">
-					<a class="ui small green button" href="{{$.Link}}/new">{{.locale.Tr "repo.projects.new"}}</a>
+		<div class="navbar gt-mb-4">
+			<div>
+				<div class="small-pill-buttons ui compact tiny menu">
```

Comment:
small-pill-buttons  ? Where is the inner button .....

Reply:
What you mean? The inner elements are two `<a>` elements in these.

Reply:
But `small-pill-buttons` seems to "make the inner buttons as small pill"?

Reply:
It gives those "buttons" consistent height as fomantic has no "small" class for these. Could add a `role=button` to them If you want that.

Reply:
That's my question, where are the buttons ... the name "small-pill-buttons" seems unclear to me, and no idea why it is necessary.

I can only see "menu items", ps: I do not think "role=button" is related.

Reply:
Maybe `small-menu-items` would be more clear. Only issue is that the "menu" term is very overloaded in fomantic context and can mean a lot of things.

Reply:
`item` is also very  overloaded in fomantic , eg: list item, etc.

So, I think it's fine to use `small-menu-items` or `g-menu-items-small` to naming it from large scope to small scope in Gitea's framework.

Reply:
I'm not a fan of either `g-` or `gt-` prefixes. As long as the class names are sufficiently unique and do not collide with other 3rd party css (besides fomantic, this could also be monaco etc), we should be free to use without prefixes.

Reply:
Renamed.